saleor/attribute/migrations/0034_assignedpageattributevalue_page_data_migration.py

```diff
@@ -1,18 +1,52 @@
 # Generated by Django 3.2.20 on 2023-08-24 11:25
 
-from django.db import migrations
-from django.db.models.signals import post_migrate
-from django.apps import apps as registry
+from django.db import migrations, transaction, connection
 
-from .tasks.saleor3_16 import assign_pages_to_attribute_values_task
 
+PAGE_BATCH_SIZE = 5000
```

Comment:
Please do not use magic numbers. Could you please provide information about ram usage on that batch size?

Reply:
https://github.com/saleor/saleor/pull/14242#discussion_r1352279239

Reply:
It's the same case as before, it's port of the changes that were moved from task